okdc-centos.sh

```diff
@@ -150,6 +164,9 @@ install_network() {
 				install_calico_with_etcd
 				;;
 			3)
+				install_calico_with_kdd
```

Comment:
Can you also add the handling for noinput mode? Just add another case in the switch above this one.

Reply:
done

Reply:
lgtm. Please squash and we will get this merged.